app/app.js

```diff
@@ -27,8 +27,13 @@ angular.module('portainer')
       originalSet.apply(cfpLoadingBar, arguments);
     }
   };
-}]);
 
+  $rootScope.$on('$viewContentLoaded', function(evt){
```

Comment:
What's the point of that code snippet?

Reply:
Since this PR replaces ContainersCount with ContainersList and adds ChildrenList, anchors are used to link the icons in the images view to the corresponding widgets in the image details view.